src/components/OverlayTooltip/index.js

```diff
@@ -17,8 +18,8 @@ const OverlayTooltip = ({ id, tooltip, placement = 'left', children, ...rest })
 }
 OverlayTooltip.propTypes = {
   id: PropTypes.string.isRequired,
-  tooltip: PropTypes.string.isRequired,
-  placement: PropTypes.string,
+  tooltip: requiredTooltipPropType,
```

Comment:
This should be inlined and referenced by other components as `Tooltip.propTypes.tooltip`